contrib/pom.xml

```diff
@@ -623,5 +628,11 @@
       <artifactId>super-csv-joda</artifactId>
       <version>2.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.gemstone.gemfire</groupId>
+      <artifactId>gemfire</artifactId>
+      <version>8.2.0</version>
+      <type>jar</type>
```

Comment:
please add `<optional>true</optional>`
`type` is by default jar so not required